src/docs/previews/table-of-contents/main/tailwind.svelte

```diff
@@ -0,0 +1,106 @@
+<script lang="ts">
+	import Tree from './tree.svelte';
+
+	import { createTableOfContents } from '@melt-ui/svelte';
+
+	const { activeHeadingIdxs, headingsTree, item } = createTableOfContents({
+		selector: '#toc-builder-preview',
+		exclude: ['h1', 'h4', 'h5', 'h6'],
+		activeType: 'lowest-parents',
+		// scrollOffset: 20,
+		// scrollBehaviour: 'smooth',
+		headingFilterFn: (heading) => !heading.hasAttribute('data-toc-ignore'),
+		scrollFn: (id) => {
+			/**
+			 * Here we're overwriting the default scroll function
+			 * so that we only scroll within the ToC preview
+			 * container, instead of the entire page.
+			 */
+			const container = document.getElementById('toc-builder-preview');
+			const element = document.getElementById(id);
+
+			if (container && element) {
+				container.scrollTo({
+					top: element.offsetTop - container.offsetTop,
+					behavior: 'smooth',
+				});
+			}
+		},
+	});
+</script>
+
+<div class="relative mx-8 w-full">
```

Comment:
The preview is not working on mobile, part of it is hidden

Reply:
@TGlide I updated the preview now. It's still a bit awkward given the lack of space, but now it's better than before